packages/message-relayer/src/exec/run.ts

```diff
@@ -14,6 +16,27 @@ const main = async () => {
   })
 
   const env = process.env
+
+  const SENTRY_DSN = config.str('use-sentry', env.SENTRY_DSN)
+  const USE_SENTRY = config.str('sentry-dsn', env.USE_SENTRY)
```

Comment:
There is a bug here in the first argument to `config.str` - they are switched. Also notice that the usage of `USE_SENTRY` is different here than in the batch submitter, code snippet below:

```js
  if (config.bool('use-sentry', env.USE_SENTRY === 'true')) {
```

It should be standardized so that the config options are similar, is just the definition of the env var enough or must it be set explicitly to `true`? I'm not super opinionated, it mostly impacts how we do our deployments, ie do we skip having the env var to turn it off or do we skip setting the env var to turn it off. Skipping the setting of the env var is less lines of code in the current way of doing deployments but being standard is most important

Reply:
Haha. Sorry.